content/app_competition_rules.tex

```diff
@@ -71,17 +85,123 @@ \subsection{Winner and Rankings}
+Note that, the penalty persists for those players penalized for ``Motion in Set''. All robots that are still penalized for other infractions will stay at the sideline until their penalty time is over.
+
+\begin{figure}[t]
+\centerline{\includegraphics[width=\columnwidth]{figs/manual-placement-cs.pdf}}
+\caption{Challenge Shield manual setup for kick-off.  The attacking team is on the right.}
+\label{fig:ko-manual}
+\end{figure}
+
+\subsubsection{Kick-In (Section \ref{sec:kick_in})}
+\label{sub:cc-kick-in}
+
+The position the ball is placed for a kick-in is modified as follows:
+\begin{itemize}
+    \item If the ball goes over a sideline, the ball is placed approximately 2-ball-widths in from the sideline perpendicular to the point where the ball went out.
+    \item If the ball goes over a sideline, closer to an end-line than the corresponding penalty spot, the ball is placed 2-ball-widths in from the sideline and in-line with the penalty spot.
+    \item For a corner kick, the ball is also placed 2-ball-widths in from the sideline and in-line with the penalty spot.
+\end{itemize}
+
+\subsubsection{Free-Kick Procedure (Section \ref{sec:free_kick})}
+\label{sub:cc-free-kick-procedure}
+
+The procedure for a \emph{Free Kick} is replaced with a procedure similar to the Penalty Free Kick.
+
+When a Free Kick is initiated the head referee will announce a Free Kick, calling one of:
+\begin{enumerate}
+  \item For a \textit{Pushing Free Kick}: ``Foul \textless color\textgreater \textless number\textgreater'' for the pushing robot.
+  \item For all other free kicks: ``Kick-in/Goal Kick/Corner Kick \textless team\textgreater'' for the team that did not last touch the ball.
+\end{enumerate}
+
+The GameController will then activate the substate for the respective free kick. Note that in the case of the Pushing Free Kick the substate is activated automatically through the ``Foul''.
+The game changes to the \textit{Ready} state.
+This denotes that the robot's are given time to setup and prepare for the free kick.
+The game clock is \textit{paused} during finals games only.
+
+The head referees \emph{immediately} places the ball on the spot for the Free Kick.
+Players have \qty{\SCFreeKickSetupTime}{\second} to get into position for the Free Kick.
+At the end of \qty{\SCFreeKickSetupTime}{\second}, the game changes to the \textit{Set} state.
+Similar to a kick-off, during the Set state the robots are waiting for the Free Kick to commence.
+Standard penalties apply.
+Additionally, no player may touch the ball and defending players may not be within \FreeKickRadius of the ball.
```

Comment:
No exception for the goalkeeper within its own goal/penalty area?

Reply:
A good point, I forgot to add that - there should be an exception for the goal keeper. I will get on this tomorrow.